scripts/add-chain.sh

```diff
@@ -78,19 +78,56 @@ genesis:
 EOF
 
 
-# add extra addresses data
+# infer appropriate L1_RPC_URL
+case $SUPERCHAIN_TARGET in
+    "mainnet")
+        L1_RPC_URL="https://ethereum-mainnet-rpc.allthatnode.com"
```

Comment:
This URL already exists in the `superchain/configs/mainnet/superchain.yaml` file. For now I will just replicate it, didn't seem worth parsing yaml in a shell script to dedupe.

Perhaps +1 point for rewriting this script in a different language if we want to do that.

Reply:
Another idea is to use [yq](https://github.com/mikefarah/yq) as a drop in replacement for jq, and it also handles YAML